packages/neuron-wallet/src/services/cells.ts

```diff
@@ -1067,9 +1071,8 @@ export default class CellsService {
         from
             output
         where
-            output.lockCodeHash = '${SystemScriptInfo.MULTI_SIGN_CODE_HASH}' AND
-            output.status in ('${OutputStatus.Live}', '${OutputStatus.Sent}') AND
-            output.lockHashType = '${SystemScriptInfo.MULTI_SIGN_HASH_TYPE}'
+            output.lockHash in (${lockHashes.map(v => `'${v}'`).join(',')}) AND
```

Comment:
For outside data source, we should use data interpolation like
```
output.lockHash IN (:...lockHashes)
...
{ lockHashes: [...data] }
```
to escape data

Reply:
`EntityManager.query` parameter is different with `SelectQueryBuilder .where`

Reply:
Could we use `escapeQueryWithParameters` to avoid this?
Ref: https://github.com/typeorm/typeorm/issues/556#issuecomment-1049587900